clusterloader2/testing/storage/pod-startup/ephemeral-volumes/config.yaml

```diff
@@ -0,0 +1,97 @@
+#Constants
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 1}}
```

Comment:
QQ, should the test scale with the size of the cluster and what was the cluster size you had in your mind when designing this test?

We're running scalability tests on clusters up to 5K nodes, if this test was run on such cluster as is, it would create 5K namespaces with 1 pod in each namespace? Is it want you wanted to achieve?

Reply:
Actually there will be three different cluster usage scenarios, 2 of them will use 1 node and one of them will be cluster wide.

There will be a [test rollout phase](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190204-scale-testing.md#test-rollout) before these tests are run regularly, to determine the thresholds, but I don't how that rollout process will be executed, before PR merge or after?, maybe @msau42 knows. I thought as these parameters can be changed manually during rollout phase, so I didn't create different override files for 1 node case and cluster wide case. Because the same thing is also valid for parameters like how many volumes will be created per node, or how many volumes will be created per node. Of course I can create different override files with the [max limits](https://kubernetes.io/docs/setup/best-practices/cluster-large/) for each volume type.

Reply:
There will be a test rollout phase before these tests are run regularly, to determine the thresholds, but I don't know how that rollout process will be executed, before PR merge or after?

We should mark this test as more experimental (maybe under experimental/ directory) and we can do that after the PR is merged. But I'm fine either way too.

@mm4tt - the first two cases, are more about performance than scalability, and will be run against very small (1-node?) cluster. That is probably important part of the answer to the question.

@mucahitkurt - I think that we should focus on the first 2 case first (as the usecase there is completely different). And leave the third case (where the tests may potentially be visibly different) as a second thing. WDYT?

Reply:
Covering only first 2 cases makes sense to me, I can create a different PR for the 3rd case after this PR is merged. I'll put these test cases under an `experimental` directory and `1_node` directory.